src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -367,7 +389,7 @@ private unsafe bool WasEventDetected(int pollFileDescriptor, int valueFileDescri
                 }
                 if (waitResult > 0)
                 {
-                    pinNumber = events.data.pinNumber;
+                    pinNumber = events.data.pinNumber - _pinOffset;
```

Comment:
>pinNumber [](start = 20, length = 9)

could you please add Assert for not getting negative values?

Reply:
sure

Reply:
turns out we don't need to subtract this value, this pin number is not set from device it is set here: https://github.com/dotnet/iot/pull/386/files/d8d60bade8e87c0d82510f04525e23656949f798#diff-f9b9a68ada4ceabc2a9b052f1c52a2cbR362 where offset address not added